source/appendices/b-changes-from-r5.rst

```diff
@@ -0,0 +1,41 @@
+
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+|clearpage|
+
+.. _appendix-b:
+
+Appendix B: Changes from Previous Releases
+==========================================
+
+Note: The main Code of Practice document takes precedence in the case of any conflicts between it and this appendix.
+
+Starting with Release 5.0.2 the COUNTER Code of Practice Release 5 is maintained in the GitHub repository `Project-Counter/cop5 <https://github.com/Project-Counter/cop5>`_, and all changes are tracked with GitHub issues and linked pull requests. The `change log <https://github.com/Project-Counter/cop5/blob/5.1/CHANGELOG.rst>`_ lists all changes by type and impact of the change.
+
+
+Primary changes between R5 and R5.1
+"""""""""""""""""""""""""""""""""""
+
+* **Item becoming the unit of reporting** - Improved OA reporting is one of the major objectives of R5.1, which requires reporting at the level of the item. This impacts on reporting of book usage: under R5.1, usage of each Book_Segment (e.g. chapter) MUST be reported as a separate item investigation or request, with multiple item investigations or requests rolled up to 1 title investigation or request per book.
+* **Data_Type** - Many publishers reported difficulties with the restricted list of Data_Types provided in R5, so the list has been expanded and the definitions and use of each Data_Type clarified.
+* **Section_Type** - The impact of the above changes means Section_Type is deprecated in R5.1.
```

Comment:
"is removed", not "is deprecated" (the latter would mean that the element would still be present, but its use discouraged)